com.vrcfury.vrcfury/Editor/VF/Service/ActionClipService.cs

```diff
@@ -83,6 +83,34 @@ public class ActionClipService {
                         }
                         break;
                     }
+                    case PoiyomiUVTileAction poiyomiUVTileAction: {
+                        var renderer = poiyomiUVTileAction.renderer;
+                        if (poiyomiUVTileAction.slot1 > 3 || poiyomiUVTileAction.slot1 < 0) {
+                            throw new ArgumentException("Poiyomi UV Tiles are ranges between 0-3, check if slots are within these ranges.");
+                        }
+                        if (poiyomiUVTileAction.slot2 > 3 || poiyomiUVTileAction.slot2 < 0) {
+                            throw new ArgumentException("Poiyomi UV Tiles are ranges between 0-3, check if slots are within these ranges.");
+                        }
+                        if (renderer != null) {
+                            var propertyName = poiyomiUVTileAction.dissolveAlpha ? "_UVTileDissolveAlpha_Row" : "_UDIMDiscardRow";
+                            if (poiyomiUVTileAction.renamedMaterial != "")
+                                propertyName += $"_{poiyomiUVTileAction.renamedMaterial}"; //Able to add renamed material if added.
+                            var binding = EditorCurveBinding.FloatCurve(
+                                clipBuilder.GetPath(renderer.gameObject),
+                                renderer.GetType(),
+                                $"material.{propertyName+poiyomiUVTileAction.slot1}_{(poiyomiUVTileAction.slot2)}"
+                            );
+                            if (!poiyomiUVTileAction.toggled)
+                                offClip.SetConstant(binding, poiyomiUVTileAction.invert ? 1f: 0f); //Ternary Operator easier to than If statement
+                            onClip.SetConstant(binding, poiyomiUVTileAction.invert ? 0f : 1f);
+                        }
+                        if (onClip != firstClip) {

```

Comment:
What's going on here?

Reply:
It's a check that I noticed in the previous clip maker, and I have it in here a just incase since I had errors being made with conflict in animation states when it makes the default state.

Reply:
this definitely should not be here

Reply:
I'm sorry if you dislike it, could you explain why it shouldn't be there?